packages/front-end/pages/settings/index.tsx

```diff
@@ -326,11 +360,14 @@ const GeneralSettingsPage = (): React.ReactElement => {
                 )}
             </div>
           </div>
-          <div className="divider border-bottom mb-3 mt-3"></div>
+
+          <div className="divider border-bottom mb-3 mt-3" />
+
           <div className="row">
             <div className="col-sm-3">
-              <h4>Other Settings</h4>
+              <h4>Experiment Settings</h4>
```

Comment:
This UI section was getting a bit big with "Other Settings" so I split it up into "Experiment Settings" and "Metrics Settings"